python/test/unit/mesh/test_mesh.py

```diff
@@ -536,3 +538,10 @@ def test_coords():
     d = mesh.coordinate_dofs().entity_points()
     d += 2
     assert numpy.array_equal(d, mesh.coordinate_dofs().entity_points())
+
+
+def test_UnitHexMesh_assemble():
+    mesh = UnitCubeMesh(MPI.comm_world, 6, 7, 5, CellType.hexahedron)
+    vol = assemble_scalar(Constant(mesh, 1) * dx)
```

Comment:
Do not complicate here with Constant, `1.0 * ufl.dx(mesh)` should be enough...

Reply:
I am also wondering if this is really a useful test. Does integrating a constant actually care about the ordering, for example.

Reply:
This test is useful. If the ordering is wrong, the integral is not 1. For instance, without this fix the integral is 0.57.